<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns:st1="urn:schemas-microsoft-com:office:smarttags"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 14">
<meta name=Originator content="Microsoft Word 14">
<link rel=File-List href="Help_Output_files/filelist.xml">
<link rel=Edit-Time-Data href="Help_Output_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]-->
<title>Output File Formats</title>
<o:SmartTagType namespaceuri="urn:schemas-microsoft-com:office:smarttags"
 name="place"/>
<!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>WFU</o:Author>
  <o:LastAuthor>Amy Olex</o:LastAuthor>
  <o:Revision>23</o:Revision>
  <o:TotalTime>139</o:TotalTime>
  <o:Created>2007-07-06T14:45:00Z</o:Created>
  <o:LastSaved>2012-06-13T20:02:00Z</o:LastSaved>
  <o:Pages>4</o:Pages>
  <o:Words>2557</o:Words>
  <o:Characters>14575</o:Characters>
  <o:Company>WFU</o:Company>
  <o:Lines>121</o:Lines>
  <o:Paragraphs>34</o:Paragraphs>
  <o:CharactersWithSpaces>17098</o:CharactersWithSpaces>
  <o:Version>14.00</o:Version>
 </o:DocumentProperties>
 <o:OfficeDocumentSettings>
  <o:DoNotRelyOnCSS/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=themeData href="Help_Output_files/themedata.thmx">
<link rel=colorSchemeMapping href="Help_Output_files/colorschememapping.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:SpellingState>Clean</w:SpellingState>
  <w:GrammarState>Clean</w:GrammarState>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-US</w:LidThemeOther>
  <w:LidThemeAsian>X-NONE</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:UseWord2002TableStyleRules/>
   <w:SplitPgBreakAndParaMark/>
  </w:Compatibility>
  <w:DocumentVariables>
   <w:EN.InstantFormat>&lt;ENInstantFormat&gt;&lt;Enabled&gt;1&lt;/Enabled&gt;&lt;ScanUnformatted&gt;1&lt;/ScanUnformatted&gt;&lt;ScanChanges&gt;1&lt;/ScanChanges&gt;&lt;/ENInstantFormat&gt;</w:EN.InstantFormat>
   <w:EN.Layout>&lt;ENLayout&gt;&lt;Style&gt;Author-Date&lt;/Style&gt;&lt;LeftDelim&gt;{&lt;/LeftDelim&gt;&lt;RightDelim&gt;}&lt;/RightDelim&gt;&lt;FontName&gt;Times<span xmlns="" style="mso-spacerun:yes">&nbsp;&nbsp;&nbsp;&nbsp; </span>
    New
    <span xmlns="" style="mso-spacerun:yes">&nbsp;&nbsp;&nbsp;&nbsp;     </span>Roman&lt;/FontName&gt;&lt;FontSize&gt;12&lt;/FontSize&gt;&lt;ReflistTitle&gt;&lt;/ReflistTitle&gt;&lt;StartingRefnum&gt;1&lt;/StartingRefnum&gt;&lt;FirstLineIndent&gt;0&lt;/FirstLineIndent&gt;&lt;HangingIndent&gt;720&lt;/HangingIndent&gt;&lt;LineSpacing&gt;0&lt;/LineSpacing&gt;&lt;SpaceAfter&gt;0&lt;/SpaceAfter&gt;&lt;HyperlinksEnabled&gt;1&lt;/HyperlinksEnabled&gt;&lt;HyperlinksVisible&gt;0&lt;/HyperlinksVisible&gt;&lt;/ENLayout&gt;</w:EN.Layout>
   <w:EN.Libraries>&lt;Libraries&gt;&lt;item<span xmlns="" style="mso-spacerun:yes">&nbsp;&nbsp;&nbsp;&nbsp;     </span>db-id=&quot;0wxpavtzjx2x9iexe5bpffz5avawff59dxsp&quot;&gt;OlexReferenceLibrary&lt;record-ids&gt;&lt;item&gt;37&lt;/item&gt;&lt;item&gt;624&lt;/item&gt;&lt;/record-ids&gt;&lt;/item&gt;&lt;/Libraries&gt;</w:EN.Libraries>
   <w:EN.Layout>&lt;ENLayout&gt;&lt;Style&gt;Author-Date&lt;/Style&gt;&lt;LeftDelim&gt;{&lt;/LeftDelim&gt;&lt;RightDelim&gt;}&lt;/RightDelim&gt;&lt;FontName&gt;Times<span xmlns="" style="mso-spacerun:yes">&nbsp;&nbsp;&nbsp;&nbsp; </span>
                                  New                              
    <span xmlns="" style="mso-spacerun:yes">&nbsp;&nbsp;&nbsp;&nbsp;     </span>Roman&lt;/FontName&gt;&lt;FontSize&gt;12&lt;/FontSize&gt;&lt;ReflistTitle&gt;&lt;/ReflistTitle&gt;&lt;StartingRefnum&gt;1&lt;/StartingRefnum&gt;&lt;FirstLineIndent&gt;0&lt;/FirstLineIndent&gt;&lt;HangingIndent&gt;720&lt;/HangingIndent&gt;&lt;LineSpacing&gt;0&lt;/LineSpacing&gt;&lt;SpaceAfter&gt;0&lt;/SpaceAfter&gt;&lt;/ENLayout&gt;</w:EN.Layout>
  </w:DocumentVariables>
  <w:BrowserLevel>MicrosoftInternetExplorer4</w:BrowserLevel>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="false"
  DefSemiHidden="false" DefQFormat="false" LatentStyleCount="267">
  <w:LsdException Locked="false" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" Priority="1" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" Priority="99" Name="No List"/>
  <w:LsdException Locked="false" Priority="99" SemiHidden="true"
   Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="99" SemiHidden="true" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" QFormat="true"
   Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" QFormat="true"
   Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" QFormat="true"
   Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" QFormat="true"
   Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" QFormat="true"
   Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" QFormat="true"
   Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" SemiHidden="true"
   UnhideWhenUsed="true" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="TOC Heading"/>
 </w:LatentStyles>
</xml><![endif]--><!--[if !mso]><object
 classid="clsid:38481807-CA0E-42D2-BF39-B33AF135CC4D" id=ieooui></object>
<style>
st1\:*{behavior:url(#ieooui) }
</style>
<![endif]-->
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Tahoma;
	panose-1:2 11 6 4 3 5 4 4 2 4;
	mso-font-charset:0;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:-520081665 -1073717157 41 0 66047 0;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0in;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";}
p.MsoHeader, li.MsoHeader, div.MsoHeader
	{mso-style-unhide:no;
	mso-style-link:"Header Char";
	margin:0in;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	tab-stops:center 3.25in right 6.5in;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";}
p.MsoFooter, li.MsoFooter, div.MsoFooter
	{mso-style-unhide:no;
	mso-style-link:"Footer Char";
	margin:0in;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	tab-stops:center 3.25in right 6.5in;
	font-size:12.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";}
p.MsoCaption, li.MsoCaption, div.MsoCaption
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-next:Normal;
	margin:0in;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Times New Roman","serif";
	mso-fareast-font-family:"Times New Roman";
	font-weight:bold;}
a:link, span.MsoHyperlink
	{mso-style-unhide:no;
	color:blue;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{mso-style-unhide:no;
	color:purple;
	text-decoration:underline;
	text-underline:single;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-unhide:no;
	mso-style-link:"Balloon Text Char";
	margin:0in;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:8.0pt;
	font-family:"Tahoma","sans-serif";
	mso-fareast-font-family:"Times New Roman";}
span.HeaderChar
	{mso-style-name:"Header Char";
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:Header;
	mso-ansi-font-size:12.0pt;
	mso-bidi-font-size:12.0pt;}
span.FooterChar
	{mso-style-name:"Footer Char";
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:Footer;
	mso-ansi-font-size:12.0pt;
	mso-bidi-font-size:12.0pt;}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Balloon Text";
	mso-ansi-font-size:8.0pt;
	mso-bidi-font-size:8.0pt;
	font-family:"Tahoma","sans-serif";
	mso-ascii-font-family:Tahoma;
	mso-hansi-font-family:Tahoma;
	mso-bidi-font-family:Tahoma;}
span.SpellE
	{mso-style-name:"";
	mso-spl-e:yes;}
span.GramE
	{mso-style-name:"";
	mso-gram-e:yes;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-size:10.0pt;
	mso-ansi-font-size:10.0pt;
	mso-bidi-font-size:10.0pt;}
 /* Page Definitions */
 @page
	{mso-footnote-separator:url("Help_Output_files/header.htm") fs;
	mso-footnote-continuation-separator:url("Help_Output_files/header.htm") fcs;
	mso-endnote-separator:url("Help_Output_files/header.htm") es;
	mso-endnote-continuation-separator:url("Help_Output_files/header.htm") ecs;}
@page WordSection1
	{size:8.5in 11.0in;
	margin:1.0in 1.25in 1.0in 1.25in;
	mso-header-margin:.5in;
	mso-footer-margin:.5in;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 @list l0
	{mso-list-id:40176515;
	mso-list-type:hybrid;
	mso-list-template-ids:-1830663018 67698689 67698691 67698693 67698689 67698691 67698693 67698689 67698691 67698693;}
@list l0:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	font-family:Symbol;}
@list l0:level2
	{mso-level-tab-stop:1.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level3
	{mso-level-tab-stop:1.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level4
	{mso-level-tab-stop:2.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level5
	{mso-level-tab-stop:2.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level6
	{mso-level-tab-stop:3.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level7
	{mso-level-tab-stop:3.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level8
	{mso-level-tab-stop:4.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level9
	{mso-level-tab-stop:4.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1
	{mso-list-id:98137195;
	mso-list-type:hybrid;
	mso-list-template-ids:552354576 67698691 67698691 67698693 67698689 67698691 67698693 67698689 67698691 67698693;}
@list l1:level1
	{mso-level-number-format:bullet;
	mso-level-text:o;
	mso-level-tab-stop:.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	font-family:"Courier New";}
@list l1:level2
	{mso-level-tab-stop:1.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level3
	{mso-level-tab-stop:1.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level4
	{mso-level-tab-stop:2.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level5
	{mso-level-tab-stop:2.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level6
	{mso-level-tab-stop:3.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level7
	{mso-level-tab-stop:3.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level8
	{mso-level-tab-stop:4.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level9
	{mso-level-tab-stop:4.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l2
	{mso-list-id:616571092;
	mso-list-type:hybrid;
	mso-list-template-ids:1170221050 67698689 67698691 67698693 67698689 67698691 67698693 67698689 67698691 67698693;}
@list l2:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	font-family:Symbol;}
@list l2:level2
	{mso-level-tab-stop:1.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l2:level3
	{mso-level-tab-stop:1.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l2:level4
	{mso-level-tab-stop:2.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l2:level5
	{mso-level-tab-stop:2.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l2:level6
	{mso-level-tab-stop:3.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l2:level7
	{mso-level-tab-stop:3.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l2:level8
	{mso-level-tab-stop:4.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l2:level9
	{mso-level-tab-stop:4.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l3
	{mso-list-id:1104422040;
	mso-list-type:hybrid;
	mso-list-template-ids:181321084 67698689 67698691 67698693 67698689 67698691 67698693 67698689 67698691 67698693;}
@list l3:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	font-family:Symbol;}
@list l3:level2
	{mso-level-tab-stop:1.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l3:level3
	{mso-level-tab-stop:1.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l3:level4
	{mso-level-tab-stop:2.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l3:level5
	{mso-level-tab-stop:2.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l3:level6
	{mso-level-tab-stop:3.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l3:level7
	{mso-level-tab-stop:3.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l3:level8
	{mso-level-tab-stop:4.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l3:level9
	{mso-level-tab-stop:4.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l4
	{mso-list-id:1888951383;
	mso-list-type:hybrid;
	mso-list-template-ids:-354099954 67698689 67698691 67698693 67698689 67698691 67698693 67698689 67698691 67698693;}
@list l4:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F0B7;
	mso-level-tab-stop:.5in;
	mso-level-number-position:left;
	text-indent:-.25in;
	font-family:Symbol;}
@list l4:level2
	{mso-level-tab-stop:1.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l4:level3
	{mso-level-tab-stop:1.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l4:level4
	{mso-level-tab-stop:2.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l4:level5
	{mso-level-tab-stop:2.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l4:level6
	{mso-level-tab-stop:3.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l4:level7
	{mso-level-tab-stop:3.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l4:level8
	{mso-level-tab-stop:4.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l4:level9
	{mso-level-tab-stop:4.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
ol
	{margin-bottom:0in;}
ul
	{margin-bottom:0in;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
 table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0in 5.4pt 0in 5.4pt;
	mso-para-margin:0in;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Times New Roman","serif";}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:shapedefaults v:ext="edit" spidmax="2049"/>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <o:shapelayout v:ext="edit">
  <o:idmap v:ext="edit" data="1"/>
 </o:shapelayout></xml><![endif]-->
</head>

<body lang=EN-US link=blue vlink=purple style='tab-interval:.5in'>

<div class=WordSection1>

<p class=MsoNormal align=center style='text-align:center'><b style='mso-bidi-font-weight:
normal'><font size=5 face="Times New Roman"><span style='font-size:16.0pt;
font-weight:bold;mso-bidi-font-weight:normal'><o:p>&nbsp;</o:p></span></font></b></p>

<p class=MsoNormal align=center style='text-align:center'><b style='mso-bidi-font-weight:
normal'><font size=5 face="Times New Roman"><span style='font-size:16.0pt;
font-weight:bold;mso-bidi-font-weight:normal'>Output File Formats for SC<sup>2</sup>ATmd
v3<o:p></o:p></span></font></b></p>

<p class=MsoNormal align=center style='text-align:center'><i style='mso-bidi-font-style:
normal'><font size=2 face="Times New Roman"><span style='font-size:10.0pt;
font-style:italic;mso-bidi-font-style:normal'>Last Updated on 6/13/2012 by Amy
Olex<o:p></o:p></span></font></i></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><i style='mso-bidi-font-style:
normal'><font size=3 face="Times New Roman"><span style='font-size:12.0pt;
font-weight:bold;mso-bidi-font-weight:normal;font-style:italic;mso-bidi-font-style:
normal'>Index<o:p></o:p></span></font></i></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><i style='mso-bidi-font-style:
normal'><font size=3 face="Times New Roman"><span style='font-size:12.0pt;
font-weight:bold;mso-bidi-font-weight:normal;font-style:italic;mso-bidi-font-style:
normal'><o:p>&nbsp;</o:p></span></font></i></b></p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='mso-list:l2 level1 lfo1;tab-stops:list .5in'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt'><a href="#fom">Figure
     of Merit Analysis</a></span></font></li>
 <li class=MsoNormal style='mso-list:l2 level1 lfo1;tab-stops:list .5in'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt'><a
     href="#cluster">Standard Clustering Analysis</a></span></font></li>
 <li class=MsoNormal style='mso-list:l2 level1 lfo1;tab-stops:list .5in'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt'><a
     href="#consensus">Consensus Clustering Analysis</a></span></font></li>
 <li class=MsoNormal style='mso-list:l2 level1 lfo1;tab-stops:list .5in'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt'><a
     href="#mapping">Cluster Mapping</a></span></font></li>
 <li class=MsoNormal style='mso-list:l2 level1 lfo1;tab-stops:list .5in'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt'><a
     href="#heatmap">Generate Heatmaps</a></span></font></li>
 <li class=MsoNormal style='mso-list:l2 level1 lfo1;tab-stops:list .5in'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt'><a
     href="#stats">Cluster Statistics</a></span></font></li>
</ul>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>This section describes the format and content of all the output files
for each analysis function. Output files come in 3 types: text files, image
files, and Matlab .fig files. </span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><i style='mso-bidi-font-style:
normal'><font size=3 face="Times New Roman"><span style='font-size:12.0pt;
font-weight:bold;mso-bidi-font-weight:normal;font-style:italic;mso-bidi-font-style:
normal'>Important Notes:<o:p></o:p></span></font></i></b></p>

<ul style='margin-top:0in' type=circle>
 <li class=MsoNormal style='mso-list:l1 level1 lfo2;tab-stops:list .5in'><b
     style='mso-bidi-font-weight:normal'><i style='mso-bidi-font-style:normal'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt;font-weight:
     bold;mso-bidi-font-weight:normal;font-style:italic;mso-bidi-font-style:
     normal'>All output will be saved in the same location as the input file
     unless the user specifies another destination.<o:p></o:p></span></font></i></b></li>
</ul>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><i style='mso-bidi-font-style:
normal'><font size=3 face="Times New Roman"><span style='font-size:12.0pt;
font-weight:bold;mso-bidi-font-weight:normal;font-style:italic;mso-bidi-font-style:
normal'><o:p>&nbsp;</o:p></span></font></i></b></p>

<div class=MsoNormal align=center style='text-align:center'><font size=3
face="Times New Roman"><span style='font-size:12.0pt'><!--[if gte vml 1]><v:shapetype
 id="_x0000_t75" coordsize="21600,21600" o:spt="75" o:preferrelative="t"
 path="m@4@5l@4@11@9@11@9@5xe" filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="_x0000_i1025" type="#_x0000_t75" style='width:427pt;
 height:3.95pt' o:hrpct="0" o:hralign="center" o:hr="t">
 <v:imagedata src="Help_Output_files/image001.png" o:title="BD14711_"/>
</v:shape><![endif]--><![if !vml]><img border=0 height=5 width=569
src="Help_Output_files/image002.gif" alt=--- v:shapes="_x0000_i1025"><![endif]></span></font></div>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><i style='mso-bidi-font-style:
normal'><font size=4 face="Times New Roman"><span style='font-size:14.0pt;
font-weight:bold;mso-bidi-font-weight:normal;font-style:italic;mso-bidi-font-style:
normal'><o:p>&nbsp;</o:p></span></font></i></b></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><i style='mso-bidi-font-style:
normal'><font size=4 face="Times New Roman"><span style='font-size:14.0pt;
font-weight:bold;mso-bidi-font-weight:normal;font-style:italic;mso-bidi-font-style:
normal'>Figure of Merit Analysis<a name=fom></a><span style='mso-tab-count:
1'>         </span></span></font></i></b><a href="#_top">top</a><b
style='mso-bidi-font-weight:normal'><i style='mso-bidi-font-style:normal'><font
size=4><span style='font-size:14.0pt;font-weight:bold;mso-bidi-font-weight:
normal;font-style:italic;mso-bidi-font-style:normal'><o:p></o:p></span></font></i></b></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>By default the figure of merit analysis produces a text file (.txt) and
an image file (.fig).<span style='mso-spacerun:yes'>  </span>The image may be
saved in additional image file formats according to user preference.<span
style='mso-spacerun:yes'>  </span>A description of each follows.</span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><span style='mso-spacerun:yes'> </span></span></font></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><font size=3
face="Times New Roman"><span style='font-size:12.0pt;font-weight:bold;
mso-bidi-font-weight:normal'>Text file:</span></font></b><i style='mso-bidi-font-style:
normal'><span style='font-style:italic;mso-bidi-font-style:normal'> myfom.txt<o:p></o:p></span></i></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>A summary of the FOM analysis is output in a text file that is similar
to Figure 1. </span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><font
size=3 face="Times New Roman"><span style='font-size:12.0pt;mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Picture_x0020_2" o:spid="_x0000_i1039" type="#_x0000_t75" alt="Description: Description: Description: FOManalysis-300genesTC2-outfile"
 style='width:297pt;height:156.75pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="Help_Output_files/image003.png" o:title=" FOManalysis-300genesTC2-outfile"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=396 height=209
src="Help_Output_files/image004.jpg"
alt="Description: Description: Description: FOManalysis-300genesTC2-outfile"
v:shapes="Picture_x0020_2"><![endif]></span></font></p>

<p class=MsoCaption align=center style='text-align:center'><b style='mso-bidi-font-weight:
normal'><font size=2 face="Times New Roman"><span style='font-size:10.0pt;
mso-bidi-font-weight:normal'>Figure </span></font></b><!--[if supportFields]><span
style='mso-element:field-begin'></span><span
style='mso-spacerun:yes'> </span>SEQ Figure \* ARABIC <span style='mso-element:
field-separator'></span><![endif]--><span style='mso-no-proof:yes'>1</span><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->: FOM text output file.</p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>The first line identifies the figure of merit version that was used. If
Euclidean distance was chosen as the similarity measure, then the original
Euclidean-biased version of the FOM will be used, else if Pearson's correlation
coefficient is used then the correlation-biased FOM will be used. The second
line reiterates the list of cluster numbers the user entered. The third line
indicates the clustering algorithm that performed the best on this data; this
is the recommended clustering algorithm. The next three sections list the
sequence of FOM scores for each clustering algorithm chosen. This is followed
by the identification of the optimal number of clusters to use with the input
data set. </span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><font size=3
face="Times New Roman"><span style='font-size:12.0pt;font-weight:bold;
mso-bidi-font-weight:normal'>Image files:</span></font></b><i style='mso-bidi-font-style:
normal'><span style='font-style:italic;mso-bidi-font-style:normal'> <span
class=SpellE>myfom.fig</span>, .jpg, .tiff, .bmp, .<span class=SpellE>eps</span>,
.<span class=SpellE>ai</span> and .<span class=SpellE>pdf</span></span></i></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><span style='mso-spacerun:yes'> </span></span></font></p>

<p class=MsoNormal><span class=GramE><font size=3 face="Times New Roman"><span
style='font-size:12.0pt'>For each iteration</span></font></span> of the FOM
algorithm the scores are plotted on a graph that is automatically saved as a
.fig file; the other image file formats may also be generated if the user
chooses.<span style='mso-spacerun:yes'>  </span>In Figure 2 the x-axis lists
the number of clusters used in each iteration of the FOM algorithm, and the
y-axis is the FOM score<span class=GramE>. </span>Each line on the plot
indicates the series of FOM scores calculated for a clustering algorithm. </p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>The .fig file can be opened and manipulated in Matlab; otherwise the
other image files can just be imported into documents as-is.</span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><font
size=3 face="Times New Roman"><span style='font-size:12.0pt;mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Picture_x0020_3" o:spid="_x0000_i1038" type="#_x0000_t75" alt="Description: Description: Description: SuppFig1"
 style='width:352.5pt;height:270.75pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="Help_Output_files/image005.png" o:title=" SuppFig1"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=470 height=361
src="Help_Output_files/image006.jpg"
alt="Description: Description: Description: SuppFig1" v:shapes="Picture_x0020_3"><![endif]></span></font></p>

<p class=MsoCaption align=center style='text-align:center'><a name="_top"></a><b
style='mso-bidi-font-weight:normal'><font size=2 face="Times New Roman"><span
style='font-size:10.0pt;mso-bidi-font-weight:normal'>Figure </span></font></b>2:
FOM graph output.</p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><font size=3
face="Times New Roman"><span style='font-size:12.0pt;font-style:italic;
mso-bidi-font-style:normal'>What is a FOM analysis?<o:p></o:p></span></font></i></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>The FOM analysis is a quantitative analysis that compares the performance
of different clustering algorithms on a set of data. The clustering method that
gets the lowest FOM score creates the most homogeneous clusters, and is
therefore the best method. The FOM analysis reveals which clustering algorithm
is best suited for each data set, and it gives a range for how many clusters
are optimal. For more details on what the FOM is see </span></font><!--[if supportFields]><span
style='mso-element:field-begin'></span><span
style='mso-spacerun:yes'> </span>ADDIN EN.CITE
&lt;EndNote&gt;&lt;Cite&gt;&lt;Author&gt;Yeung&lt;/Author&gt;&lt;Year&gt;2001&lt;/Year&gt;&lt;RecNum&gt;37&lt;/RecNum&gt;&lt;DisplayText&gt;(Yeung,
Haynor et al. 2001)&lt;/DisplayText&gt;&lt;record&gt;&lt;rec-number&gt;37&lt;/rec-number&gt;&lt;foreign-keys&gt;&lt;key
app=&quot;EN&quot;
db-id=&quot;0wxpavtzjx2x9iexe5bpffz5avawff59dxsp&quot;&gt;37&lt;/key&gt;&lt;/foreign-keys&gt;&lt;ref-type
name=&quot;Journal Article&quot;&gt;17&lt;/ref-type&gt;&lt;contributors&gt;&lt;authors&gt;&lt;author&gt;Yeung,
K. Y.&lt;/author&gt;&lt;author&gt;Haynor, D. R.&lt;/author&gt;&lt;author&gt;Ruzzo,
W.
L.&lt;/author&gt;&lt;/authors&gt;&lt;/contributors&gt;&lt;auth-address&gt;Computer
Science and Engineering, Box 352350, University of Washington, Seattle, WA
98195, USA.&lt;/auth-address&gt;&lt;titles&gt;&lt;title&gt;Validating
clustering for gene expression data&lt;/title&gt;&lt;secondary-title&gt;Bioinformatics&lt;/secondary-title&gt;&lt;/titles&gt;&lt;periodical&gt;&lt;full-title&gt;Bioinformatics&lt;/full-title&gt;&lt;/periodical&gt;&lt;pages&gt;309-18&lt;/pages&gt;&lt;volume&gt;17&lt;/volume&gt;&lt;number&gt;4&lt;/number&gt;&lt;keywords&gt;&lt;keyword&gt;*Algorithms&lt;/keyword&gt;&lt;keyword&gt;Animals&lt;/keyword&gt;&lt;keyword&gt;Barrett
Esophagus/genetics&lt;/keyword&gt;&lt;keyword&gt;Central Nervous
System&lt;/keyword&gt;&lt;keyword&gt;*Databases,
Factual&lt;/keyword&gt;&lt;keyword&gt;Female&lt;/keyword&gt;&lt;keyword&gt;*Gene
Expression&lt;/keyword&gt;&lt;keyword&gt;Humans&lt;/keyword&gt;&lt;keyword&gt;Ovary&lt;/keyword&gt;&lt;keyword&gt;Rats&lt;/keyword&gt;&lt;keyword&gt;Saccharomyces
cerevisiae&lt;/keyword&gt;&lt;keyword&gt;*Software
Validation&lt;/keyword&gt;&lt;/keywords&gt;&lt;dates&gt;&lt;year&gt;2001&lt;/year&gt;&lt;pub-dates&gt;&lt;date&gt;Apr&lt;/date&gt;&lt;/pub-dates&gt;&lt;/dates&gt;&lt;accession-num&gt;11301299&lt;/accession-num&gt;&lt;label&gt;yeung2001&lt;/label&gt;&lt;urls&gt;&lt;related-urls&gt;&lt;url&gt;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;amp;db=PubMed&amp;amp;dopt=Citation&amp;amp;list_uids=11301299&lt;/url&gt;&lt;/related-urls&gt;&lt;/urls&gt;&lt;/record&gt;&lt;/Cite&gt;&lt;/EndNote&gt;<span
style='mso-element:field-separator'></span><![endif]--><span style='mso-no-proof:
yes'>(<a href="#_ENREF_2" title="Yeung, 2001 #37"><font color=black><span
style='color:windowtext;text-decoration:none;text-underline:none'>Yeung, Haynor
et al. 2001</span></font></a>)</span><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->. </p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><font size=3
face="Times New Roman"><span style='font-size:12.0pt;font-style:italic;
mso-bidi-font-style:normal'>Determining the Optimal Clustering Method<o:p></o:p></span></font></i></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>The lower the FOM score, the better; therefore the line closest to the
bottom of the graph is the optimal clustering method. This program calculates
the average FOM score for each method, and the method with the lower average is
chosen as optimal; this information can be found in the text output file
discussed previously. </span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><font size=3
face="Times New Roman"><span style='font-size:12.0pt;font-style:italic;
mso-bidi-font-style:normal'>Choosing the Optimal Number of Clusters<o:p></o:p></span></font></i></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>The number of clusters is chosen based on where the 'elbow' in the
graph is. This 'elbow' indicates that increasing the number of clusters is not
improving the overall cluster homogeneity, so the FOM score is not improving
much and is flattening out. The optimal number of clusters is indicated in the
text summary file; any <span class=GramE>number</span> of clusters that fall in
the enclosed range are acceptable.<span style='mso-spacerun:yes'>  </span></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><i style='mso-bidi-font-style:normal'><font size=3
face="Times New Roman"><span style='font-size:12.0pt;font-style:italic;
mso-bidi-font-style:normal'>Customizable Graph Features (Matlab users)<o:p></o:p></span></font></i></p>

<p class=MsoNormal><span class=SpellE><font size=3 face="Times New Roman"><span
style='font-size:12.0pt'>Matlab's</span></font></span> .fig file gives the user
the capability to customize the look of the FOM plot. Matlab tutorials can be
found on the web; below is a short, non-comprehensive list of customizable
graph features. </p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='mso-list:l0 level1 lfo3;tab-stops:list .5in'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt'>Font and font
     size </span></font></li>
 <li class=MsoNormal style='mso-list:l0 level1 lfo3;tab-stops:list .5in'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt'>Grid lines
     on/off </span></font></li>
 <li class=MsoNormal style='mso-list:l0 level1 lfo3;tab-stops:list .5in'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt'>Axis labels </span></font></li>
 <li class=MsoNormal style='mso-list:l0 level1 lfo3;tab-stops:list .5in'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt'>Title and
     Legend </span></font></li>
 <li class=MsoNormal style='mso-list:l0 level1 lfo3;tab-stops:list .5in'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt'>Axis markers </span></font></li>
 <li class=MsoNormal style='mso-list:l0 level1 lfo3;tab-stops:list .5in'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt'>Line color,
     shape and size </span></font></li>
 <li class=MsoNormal style='mso-list:l0 level1 lfo3;tab-stops:list .5in'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt'>Data marker
     color, shape and size </span></font></li>
 <li class=MsoNormal style='mso-list:l0 level1 lfo3;tab-stops:list .5in'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt'>Background
     color </span></font></li>
 <li class=MsoNormal style='mso-list:l0 level1 lfo3;tab-stops:list .5in'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt'>Graph
     dimensions </span></font></li>
</ul>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<div class=MsoNormal align=center style='text-align:center'><font size=3
face="Times New Roman"><span style='font-size:12.0pt'><!--[if gte vml 1]><v:shape
 id="_x0000_i1026" type="#_x0000_t75" style='width:427pt;height:3.95pt'
 o:hrpct="0" o:hralign="center" o:hr="t">
 <v:imagedata src="Help_Output_files/image001.png" o:title="BD14711_"/>
</v:shape><![endif]--><![if !vml]><img border=0 height=5 width=569
src="Help_Output_files/image002.gif" alt=--- v:shapes="_x0000_i1026"><![endif]></span></font></div>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><i style='mso-bidi-font-style:
normal'><font size=4 face="Times New Roman"><span style='font-size:14.0pt;
font-weight:bold;mso-bidi-font-weight:normal;font-style:italic;mso-bidi-font-style:
normal'>Standard Clustering Analysis<a name=cluster></a><span style='mso-tab-count:
1'>  </span></span></font></i></b><a href="#_top">top</a><b style='mso-bidi-font-weight:
normal'><i style='mso-bidi-font-style:normal'><font size=4><span
style='font-size:14.0pt;font-weight:bold;mso-bidi-font-weight:normal;
font-style:italic;mso-bidi-font-style:normal'><o:p></o:p></span></font></i></b></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>This section describes the output generated by the standard clustering
analysis<span class=GramE>. </span>One text output file is generated automatically,
and the user has the option of also creating a text file containing the cluster
statistics, and image files representing each cluster as a heatmap.</span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>The standard clustering algorithm performs the selected clustering
method on the loaded data set once, and generates a text file with all the
clustering results.<span style='mso-spacerun:yes'>  </span>If the ‘Generate
Heatmap’ option is chosen, one heatmap with dendrogram will be generated for
each cluster and saved as a Matlab .fig file; other image file formats may also
be selected. If hierarchical clustering is selected a global dendrogram is
output in addition to the heatmaps as a .fig and .jpeg file; this dendrogram
relates each cluster to the others so that the entire hierarchical tree can be
reconstructed if desired. </span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><font size=3
face="Times New Roman"><span style='font-size:12.0pt;font-weight:bold;
mso-bidi-font-weight:normal'>Text file: </span></font></b><i style='mso-bidi-font-style:
normal'><span style='font-style:italic;mso-bidi-font-style:normal'>myclusters.txt</span></i></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>Figure 3 is an example of the clustering output file if heatmaps are
also generated.<span style='mso-spacerun:yes'>  </span>If heatmaps are not
generated, then the second column, ‘<span class=SpellE>clusterOrder</span>’
will not be included. </span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><font
size=3 face="Times New Roman"><span style='font-size:12.0pt;mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Picture_x0020_5" o:spid="_x0000_i1037" type="#_x0000_t75" alt="Description: Description: Description: StdClustTXT"
 style='width:315pt;height:244.5pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="Help_Output_files/image007.png" o:title=" StdClustTXT"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=420 height=326
src="Help_Output_files/image008.jpg"
alt="Description: Description: Description: StdClustTXT" v:shapes="Picture_x0020_5"><![endif]></span></font></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><font
size=3 face="Times New Roman"><span style='font-size:12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoCaption align=center style='text-align:center'><b style='mso-bidi-font-weight:
normal'><font size=2 face="Times New Roman"><span style='font-size:10.0pt;
mso-bidi-font-weight:normal'>Figure </span></font></b>3: Clustering text output
file.</p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><span class=GramE><b style='mso-bidi-font-weight:normal'><font
size=3 face="Times New Roman"><span style='font-size:12.0pt;font-weight:bold;
mso-bidi-font-weight:normal'>Text file</span></font></b></span><b
style='mso-bidi-font-weight:normal'><span style='font-weight:bold;mso-bidi-font-weight:
normal'> description: <o:p></o:p></span></b></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='mso-list:l3 level1 lfo4;tab-stops:list .5in'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt'>Column
     1:<span style='mso-spacerun:yes'>  </span>This column contains the cluster
     assignments for each gene in the file. </span></font></li>
 <li class=MsoNormal style='mso-list:l3 level1 lfo4;tab-stops:list .5in'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt'>Column
     2:<span style='mso-spacerun:yes'>  </span>This column contains the order
     of genes in each cluster on the heatmaps.</span></font></li>
 <li class=MsoNormal style='mso-list:l3 level1 lfo4;tab-stops:list .5in'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt'>Column
     3:<span style='mso-spacerun:yes'>  </span>The unique gene labels provided
     by the user. </span></font></li>
 <li class=MsoNormal style='mso-list:l3 level1 lfo4;tab-stops:list .5in'><font
     size=3 face="Times New Roman"><span style='font-size:12.0pt'>Columns
     4-n:<span style='mso-spacerun:yes'>  </span>The imported data for each
     gene that was used to generate the clusters. </span></font></li>
</ul>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><span class=SpellE><b style='mso-bidi-font-weight:normal'><font
size=3 face="Times New Roman"><span style='font-size:12.0pt;font-weight:bold;
mso-bidi-font-weight:normal'>Heatmap</span></font></b></span><b
style='mso-bidi-font-weight:normal'><span style='font-weight:bold;mso-bidi-font-weight:
normal'> Image files: </span></b><span class=SpellE><i style='mso-bidi-font-style:
normal'><span style='font-style:italic;mso-bidi-font-style:normal'>myclusters-clusterX.fig</span></i></span><i
style='mso-bidi-font-style:normal'><span style='font-style:italic;mso-bidi-font-style:
normal'> </span></i></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>The clustering algorithms provide the option to generate one
hierarchically clustered heatmap for each cluster.<span
style='mso-spacerun:yes'>  </span>The image files may be used as-is, or the
Matlab .fig file may be customized by advanced Matlab users. An example heatmap
and dendrogram are shown in Figure 4. </span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><font
size=3 face="Times New Roman"><span style='font-size:12.0pt;mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Picture_x0020_6" o:spid="_x0000_i1036" type="#_x0000_t75" alt="Description: Description: Description: myclusters-cluster1"
 style='width:234pt;height:210.75pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="Help_Output_files/image009.png" o:title=" myclusters-cluster1"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=312 height=281
src="Help_Output_files/image010.jpg"
alt="Description: Description: Description: myclusters-cluster1" v:shapes="Picture_x0020_6"><![endif]></span></font></p>

<p class=MsoCaption align=center style='text-align:center'><b style='mso-bidi-font-weight:
normal'><font size=2 face="Times New Roman"><span style='font-size:10.0pt;
mso-bidi-font-weight:normal'>Figure </span></font></b>4: Clustering heatmap
output.</p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>Each column of the heatmap represents an experimental condition, and
each row is one gene. Column labels are located at the bottom, row labels on
the right, and the file name is at the top. The dendrogram to the left was
generated by re-clustering each cluster using HAC and Euclidean distance in
order to organize each het map.<span style='mso-spacerun:yes'>  </span>This
way, each gene can be related to the others that are within the same cluster
where the height of each branch indicates how similar two genes or groups of
genes are (shorter branches indicate a stronger similarity).<span
style='mso-spacerun:yes'>  </span>The color index is located on the right;
either a red-green, yellow-blue or red-white-blue color map can be selected
where red/yellow indicate an increase from the control, green/blue represents a
decrease from the control, and black/white is no change. </span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><font size=3
face="Times New Roman"><span style='font-size:12.0pt;font-weight:bold;
mso-bidi-font-weight:normal'>Global Dendrogram files: </span></font></b><span
class=SpellE><i style='mso-bidi-font-style:normal'><span style='font-style:
italic;mso-bidi-font-style:normal'>myclusters-gden.fig</span></i></span><i
style='mso-bidi-font-style:normal'><span style='font-style:italic;mso-bidi-font-style:
normal'> and myclusters-gden.jpg</span></i></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>This unique implementation of the traditional hierarchical clustering
algorithm results in the complete hierarchical tree being divided into the
pre-selected number of clusters. To relate the individual clusters, the 'top'
of the complete dendrogram is saved as a .fig and jpeg file, so that the user
is able to relate each reported cluster and reconstruct the entire tree. An
example dendrogram is shown in Figure 5 where each numbered leaf represents a
cluster of genes for which a heatmap was created as above. </span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><font
size=3 face="Times New Roman"><span style='font-size:12.0pt;mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Picture_x0020_7" o:spid="_x0000_i1035" type="#_x0000_t75" alt="Description: Description: Description: myclusters-gden"
 style='width:234pt;height:190.5pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="Help_Output_files/image011.png" o:title=" myclusters-gden"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=312 height=254
src="Help_Output_files/image012.jpg"
alt="Description: Description: Description: myclusters-gden" v:shapes="Picture_x0020_7"><![endif]></span></font></p>

<p class=MsoCaption align=center style='text-align:center'><b style='mso-bidi-font-weight:
normal'><font size=2 face="Times New Roman"><span style='font-size:10.0pt;
mso-bidi-font-weight:normal'>Figure </span></font></b>5: Hierarchical
clustering global dendrogram output.</p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<div class=MsoNormal align=center style='text-align:center'><font size=3
face="Times New Roman"><span style='font-size:12.0pt'><!--[if gte vml 1]><v:shape
 id="_x0000_i1027" type="#_x0000_t75" style='width:427pt;height:3.95pt'
 o:hrpct="0" o:hralign="center" o:hr="t">
 <v:imagedata src="Help_Output_files/image001.png" o:title="BD14711_"/>
</v:shape><![endif]--><![if !vml]><img border=0 height=5 width=569
src="Help_Output_files/image002.gif" alt=--- v:shapes="_x0000_i1027"><![endif]></span></font></div>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><a name=consensus></a><b style='mso-bidi-font-weight:normal'><i
style='mso-bidi-font-style:normal'><font size=4 face="Times New Roman"><span
style='font-size:14.0pt;font-weight:bold;mso-bidi-font-weight:normal;
font-style:italic;mso-bidi-font-style:normal'>Consensus Clustering Analysis<span
style='mso-tab-count:1'>         </span></span></font></i></b><a href="#_top">top</a><b
style='mso-bidi-font-weight:normal'><i style='mso-bidi-font-style:normal'><font
size=4><span style='font-size:14.0pt;font-weight:bold;mso-bidi-font-weight:
normal;font-style:italic;mso-bidi-font-style:normal'><o:p></o:p></span></font></i></b></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>This section describes the output generated by the consensus clustering
analysis, which is similar to the standard clustering output except for 2
additional files.<span style='mso-spacerun:yes'>  </span>Two text output files
are generated automatically, and the user has the option of also creating a
text file containing the cluster statistics, and image files representing each
cluster as a <span class=SpellE>heatmap</span>.</span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>The consensus clustering algorithm performs the selected type of
consensus clustering (see Tutorial), and generates a text file with all
clustering results in the same format as that shown in Figure 3.<span
style='mso-spacerun:yes'>  </span>If the ‘Generate <span class=SpellE>Heatmap</span>’
option is chosen, one <span class=SpellE>heatmap</span> for each cluster of
appropriate size (see Tutorial) will be generated and saved as a <span
class=SpellE>Matlab</span> .fig file; other image file formats may also be
chosen (see Figure 4).<span style='mso-spacerun:yes'>  </span></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><font size=3
face="Times New Roman"><span style='font-size:12.0pt;font-weight:bold;
mso-bidi-font-weight:normal'>Text file: </span></font></b><i style='mso-bidi-font-style:
normal'><span style='font-style:italic;mso-bidi-font-style:normal'>myclusters-ClusterRuns.txt</span></i></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><span class=GramE><font size=3 face="Times New Roman"><span
style='font-size:12.0pt'>Consensus clustering also lets the user export the
multiple clustering runs used to extract the consensus clusters to a text file</span></font></span>
(Figure 6).<span style='mso-spacerun:yes'>  </span>This is helpful if the raw
clustering data ever needs to be re-analyzed.<span style='mso-spacerun:yes'> 
</span>The first column always contains the gene IDs, followed by the
clustering solutions.<span style='mso-spacerun:yes'>  </span>In Figure 6, 2
data sets were clustered (dset1 and dset2) 3 times each (rep1, rep2 and rep3)
using <span class=SpellE>Kmeans</span> and Euclidean distance (ED) into 8
clusters.<span style='mso-spacerun:yes'>  </span>For each gene, the cluster
number it was assigned to is indicated for each clustering solution
column.<span style='mso-spacerun:yes'>  </span>This is the actual data used to
calculate the consensus clusters.<span style='mso-spacerun:yes'>  </span></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal align=center style='text-align:center'><font size=3
face="Times New Roman"><span style='font-size:12.0pt;mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Picture_x0020_1" o:spid="_x0000_i1034" type="#_x0000_t75" style='width:327pt;
 height:197.25pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="Help_Output_files/image013.png" o:title="" croptop="13550f"
  cropbottom="6084f" cropright="36646f"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=436 height=263
src="Help_Output_files/image014.jpg" v:shapes="Picture_x0020_1"><![endif]></span></font></p>

<p class=MsoNormal align=center style='text-align:center'><b style='mso-bidi-font-weight:
normal'><font size=2 face="Times New Roman"><span style='font-size:10.0pt;
mso-bidi-font-size:12.0pt;font-weight:bold;mso-bidi-font-weight:normal'>Figure
6: Example of consensus clustering run output.<o:p></o:p></span></font></b></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><font size=3
face="Times New Roman"><span style='font-size:12.0pt;font-weight:bold;
mso-bidi-font-weight:normal'>Text file: </span></font></b><i style='mso-bidi-font-style:
normal'><span style='font-style:italic;mso-bidi-font-style:normal'>myclusters_network.txt</span></i></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>Finally, each consensus clustering run automatically exports a
tab-delimited file containing data for visualizing the clusters as a network
(Figure 7, see Tutorial for details), which is a new feature in version 3.<span
style='mso-spacerun:yes'>  </span>Each row of data represents an edge in the
network, and ALL POSSIBLE edges are included in the file and must be filtered
to obtain clusters (see Tutorial).<span style='mso-spacerun:yes'> 
</span>Column <span class=GramE>A</span> (ID1) and Column B (ID2) indicate the
two genes connected by an edge with the edges attribute ‘<span class=SpellE>NumClusteredTogether</span>’.<span
style='mso-spacerun:yes'>  </span>The edge attribute is important as it tells
you how many times those two genes were found in the same cluster.<span
style='mso-spacerun:yes'>  </span>The data shown in Figure 7 was generated from
the same data in Figure 6; thus, 6 total clustering solutions were run, so the
maximum number of times 2 genes can be found in the same cluster is 6—which
represents 100% for this data.<span style='mso-spacerun:yes'>  </span>If one
were to have more clustering solutions, like 30, then two genes would have to
be found in the same cluster 30 times to be considered at the 100% threshold
(see Tutorial for details).<span style='mso-spacerun:yes'>  </span>To view this
file as a network, simply import it into a network visualization software like
Cytoscape </span></font><!--[if supportFields]><span style='mso-element:field-begin'></span><span
style='mso-spacerun:yes'> </span>ADDIN EN.CITE
&lt;EndNote&gt;&lt;Cite&gt;&lt;Author&gt;Smoot&lt;/Author&gt;&lt;Year&gt;2011&lt;/Year&gt;&lt;RecNum&gt;624&lt;/RecNum&gt;&lt;DisplayText&gt;(Smoot,
Ono et al.
2011)&lt;/DisplayText&gt;&lt;record&gt;&lt;rec-number&gt;624&lt;/rec-number&gt;&lt;foreign-keys&gt;&lt;key
app=&quot;EN&quot;
db-id=&quot;0wxpavtzjx2x9iexe5bpffz5avawff59dxsp&quot;&gt;624&lt;/key&gt;&lt;/foreign-keys&gt;&lt;ref-type
name=&quot;Journal Article&quot;&gt;17&lt;/ref-type&gt;&lt;contributors&gt;&lt;authors&gt;&lt;author&gt;Smoot,
M. E.&lt;/author&gt;&lt;author&gt;Ono,
K.&lt;/author&gt;&lt;author&gt;Ruscheinski,
J.&lt;/author&gt;&lt;author&gt;Wang, P. L.&lt;/author&gt;&lt;author&gt;Ideker,
T.&lt;/author&gt;&lt;/authors&gt;&lt;/contributors&gt;&lt;auth-address&gt;Department
of Medicine, University of California, San Diego, 9500 Gilman Drive, La Jolla,
CA 92093, USA.
msmoot@ucsd.edu&lt;/auth-address&gt;&lt;titles&gt;&lt;title&gt;Cytoscape 2.8:
new features for data integration and network
visualization&lt;/title&gt;&lt;secondary-title&gt;Bioinformatics&lt;/secondary-title&gt;&lt;/titles&gt;&lt;periodical&gt;&lt;full-title&gt;Bioinformatics&lt;/full-title&gt;&lt;/periodical&gt;&lt;pages&gt;431-2&lt;/pages&gt;&lt;volume&gt;27&lt;/volume&gt;&lt;number&gt;3&lt;/number&gt;&lt;edition&gt;2010/12/15&lt;/edition&gt;&lt;keywords&gt;&lt;keyword&gt;Computational
Biology/*methods&lt;/keyword&gt;&lt;keyword&gt;*Software&lt;/keyword&gt;&lt;keyword&gt;Statistics
as Topic/*methods&lt;/keyword&gt;&lt;/keywords&gt;&lt;dates&gt;&lt;year&gt;2011&lt;/year&gt;&lt;pub-dates&gt;&lt;date&gt;Feb
1&lt;/date&gt;&lt;/pub-dates&gt;&lt;/dates&gt;&lt;isbn&gt;1367-4811
(Electronic)&amp;#xD;1367-4803
(Linking)&lt;/isbn&gt;&lt;accession-num&gt;21149340&lt;/accession-num&gt;&lt;urls&gt;&lt;related-urls&gt;&lt;url&gt;http://www.ncbi.nlm.nih.gov/pubmed/21149340&lt;/url&gt;&lt;/related-urls&gt;&lt;/urls&gt;&lt;custom2&gt;3031041&lt;/custom2&gt;&lt;electronic-resource-num&gt;btq675
[pii]&amp;#xD;10.1093/bioinformatics/btq675&lt;/electronic-resource-num&gt;&lt;language&gt;eng&lt;/language&gt;&lt;/record&gt;&lt;/Cite&gt;&lt;/EndNote&gt;<span
style='mso-element:field-separator'></span><![endif]--><span style='mso-no-proof:
yes'>(<a href="#_ENREF_1" title="Smoot, 2011 #624"><font color=black><span
style='color:windowtext;text-decoration:none;text-underline:none'>Smoot, Ono et
al. 2011</span></font></a>)</span><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->.<span
style='mso-spacerun:yes'>   </span></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal align=center style='text-align:center'><font size=3
face="Times New Roman"><span style='font-size:12.0pt;mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Picture_x0020_4" o:spid="_x0000_i1033" type="#_x0000_t75" style='width:232.5pt;
 height:199.5pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="Help_Output_files/image015.png" o:title="" croptop="13826f"
  cropbottom="6637f" cropright="45576f"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=310 height=266
src="Help_Output_files/image016.jpg" v:shapes="Picture_x0020_4"><![endif]></span></font></p>

<p class=MsoNormal align=center style='text-align:center'><b style='mso-bidi-font-weight:
normal'><font size=2 face="Times New Roman"><span style='font-size:10.0pt;
mso-bidi-font-size:12.0pt;font-weight:bold;mso-bidi-font-weight:normal'>Figure
7: Example of cluster network output file.<o:p></o:p></span></font></b></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<div class=MsoNormal align=center style='text-align:center'><font size=3
face="Times New Roman"><span style='font-size:12.0pt'><!--[if gte vml 1]><v:shape
 id="_x0000_i1028" type="#_x0000_t75" style='width:427pt;height:3.95pt'
 o:hrpct="0" o:hralign="center" o:hr="t">
 <v:imagedata src="Help_Output_files/image001.png" o:title="BD14711_"/>
</v:shape><![endif]--><![if !vml]><img border=0 height=5 width=569
src="Help_Output_files/image002.gif" alt=--- v:shapes="_x0000_i1028"><![endif]></span></font></div>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><i style='mso-bidi-font-style:
normal'><font size=4 face="Times New Roman"><span style='font-size:14.0pt;
font-weight:bold;mso-bidi-font-weight:normal;font-style:italic;mso-bidi-font-style:
normal'>Cluster Mapping Output<a name=mapping></a><span style='mso-tab-count:
1'>         </span></span></font></i></b><a href="#_top">top</a><b
style='mso-bidi-font-weight:normal'><i style='mso-bidi-font-style:normal'><font
size=4><span style='font-size:14.0pt;font-weight:bold;mso-bidi-font-weight:
normal;font-style:italic;mso-bidi-font-style:normal'><o:p></o:p></span></font></i></b></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>The cluster mapping function outputs one tab-delimited text file that
describes one clustering solution in terms of another. An example output file
is shown in Figure 8</span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><font
size=3 face="Times New Roman"><span style='font-size:12.0pt;mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Picture_x0020_9" o:spid="_x0000_i1032" type="#_x0000_t75" alt="Description: Description: Description: userfig19"
 style='width:171pt;height:57.75pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="Help_Output_files/image017.png" o:title=" userfig19"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=228 height=77
src="Help_Output_files/image018.jpg"
alt="Description: Description: Description: userfig19" v:shapes="Picture_x0020_9"><![endif]></span></font></p>

<p class=MsoCaption align=center style='text-align:center'><b style='mso-bidi-font-weight:
normal'><font size=2 face="Times New Roman"><span style='font-size:10.0pt;
mso-bidi-font-weight:normal'>Figure </span></font></b>8: Cluster <span
class=GramE>Mapping</span> text output file.</p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>The first column lists each cluster in the first solution. The rest of
the columns list how many genes in each clustering solution1 are located in the
corresponding clustering solution2. The file can be read as follows: the
solution1 cluster #1 is composed of 21 genes from the solution2 cluster #2; the
solution1 cluster #2 is composed of 6 genes from the solution2 cluster #1 and
153 genes from the solution2 cluster #4; the solution1 cluster 3 is composed of
135 genes from the solution2 cluster #1 and 4 genes from the solution2 cluster
#3; and the solution1 cluster #4 is composed of 34 genes in solution2 cluster
#3.</span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<div class=MsoNormal align=center style='text-align:center'><font size=3
face="Times New Roman"><span style='font-size:12.0pt'><!--[if gte vml 1]><v:shape
 id="_x0000_i1029" type="#_x0000_t75" style='width:427pt;height:3.95pt'
 o:hrpct="0" o:hralign="center" o:hr="t">
 <v:imagedata src="Help_Output_files/image001.png" o:title="BD14711_"/>
</v:shape><![endif]--><![if !vml]><img border=0 height=5 width=569
src="Help_Output_files/image002.gif" alt=--- v:shapes="_x0000_i1029"><![endif]></span></font></div>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><i style='mso-bidi-font-style:
normal'><font size=4 face="Times New Roman"><span style='font-size:14.0pt;
font-weight:bold;mso-bidi-font-weight:normal;font-style:italic;mso-bidi-font-style:
normal'>Generate Heatmap Output<a name=heatmap></a><span style='mso-tab-count:
1'>      </span></span></font></i></b><a href="#_top">top</a></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>The heatmap generation function outputs <i style='mso-bidi-font-style:
normal'><span style='font-style:italic;mso-bidi-font-style:normal'>n </span></i>.fig
files, one for each of the <i style='mso-bidi-font-style:normal'><span
style='font-style:italic;mso-bidi-font-style:normal'>n </span></i>clusters that
are specified in the input file; other image file formats may also be chosen as
well as the minimum cluster size (see Tutorial). These files are the same as
those output by the standard clustering function, only the input clusters are
pre-defined.<span style='mso-spacerun:yes'>  </span>Also, a text output file,
similar to that shown in Figure 3 is output.</span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<div class=MsoNormal align=center style='text-align:center'><font size=3
face="Times New Roman"><span style='font-size:12.0pt'><!--[if gte vml 1]><v:shape
 id="_x0000_i1030" type="#_x0000_t75" style='width:427pt;height:3.95pt'
 o:hrpct="0" o:hralign="center" o:hr="t">
 <v:imagedata src="Help_Output_files/image001.png" o:title="BD14711_"/>
</v:shape><![endif]--><![if !vml]><img border=0 height=5 width=569
src="Help_Output_files/image002.gif" alt=--- v:shapes="_x0000_i1030"><![endif]></span></font></div>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><i style='mso-bidi-font-style:
normal'><font size=4 face="Times New Roman"><span style='font-size:14.0pt;
font-weight:bold;mso-bidi-font-weight:normal;font-style:italic;mso-bidi-font-style:
normal'>Cluster Statistics Output<a name=stats></a><span style='mso-tab-count:
1'> </span></span></font></i></b><a href="#_top">top</a></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>The <span class=SpellE>ClusterStats</span> function outputs one text
file containing information about each cluster<span class=GramE>. </span>An example
file is shown in Figure 9.</span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><font
size=3 face="Times New Roman"><span style='font-size:12.0pt;mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Picture_x0020_12" o:spid="_x0000_i1031" type="#_x0000_t75" alt="Description: Description: Description: Output_fig7"
 style='width:286.5pt;height:265.5pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="Help_Output_files/image019.jpg" o:title=" Output_fig7"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=382 height=354
src="Help_Output_files/image020.jpg"
alt="Description: Description: Description: Output_fig7" v:shapes="Picture_x0020_12"><![endif]></span></font></p>

<p class=MsoNormal align=center style='text-align:center;page-break-after:avoid'><font
size=3 face="Times New Roman"><span style='font-size:12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoCaption align=center style='text-align:center'><b style='mso-bidi-font-weight:
normal'><font size=2 face="Times New Roman"><span style='font-size:10.0pt;
mso-bidi-font-weight:normal'>Figure </span></font></b>9: Cluster Statistics
text output file.</p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>The very first section is a global summary of the clusters contained in
this analysis.</span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'>A description of each per-cluster element in the output file is below:</span></font></p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='mso-list:l4 level1 lfo5;tab-stops:list .5in'><i
     style='mso-bidi-font-style:normal'><font size=3 face="Times New Roman"><span
     style='font-size:12.0pt;font-style:italic;mso-bidi-font-style:normal'>Average
     Euclidean Distance Score:</span></font></i><span
     style='mso-spacerun:yes'>  </span>This score reflects the overall
     homogeneity of each cluster with respect to Euclidean distance (ED). ED
     looks for clusters that have highly similar levels of expression; thus, a
     lower ED score indicates higher homogeneity with respect to similar
     expression levels. Note that if the clusters were generated using
     Pearson's correlation coefficient then the ED score will most likely be
     high since correlation clusters are not homogeneous with respect to
     Euclidean distance.</li>
 <li class=MsoNormal style='mso-list:l4 level1 lfo5;tab-stops:list .5in'><i
     style='mso-bidi-font-style:normal'><font size=3 face="Times New Roman"><span
     style='font-size:12.0pt;font-style:italic;mso-bidi-font-style:normal'>Number
     of genes in cluster:</span></font></i><span style='mso-spacerun:yes'> 
     </span>The size of the cluster.</li>
 <li class=MsoNormal style='mso-list:l4 level1 lfo5;tab-stops:list .5in'><i
     style='mso-bidi-font-style:normal'><font size=3 face="Times New Roman"><span
     style='font-size:12.0pt;font-style:italic;mso-bidi-font-style:normal'>Average
     Expression Profile:</span></font></i><span style='mso-spacerun:yes'> 
     </span>The expression values in each condition/column were averaged to
     obtain an average profile for each cluster.</li>
 <li class=MsoNormal style='mso-list:l4 level1 lfo5;tab-stops:list .5in'><i
     style='mso-bidi-font-style:normal'><font size=3 face="Times New Roman"><span
     style='font-size:12.0pt;font-style:italic;mso-bidi-font-style:normal'>Standard
     Deviations:</span></font></i><span style='mso-spacerun:yes'>  </span>The
     standard deviation for each condition/column in a cluster is calculated.
     This is a measure of the variance at each time point. <st1:place w:st="on">Lower
      <span class=SpellE>StdDev</span></st1:place> <span class=GramE>indicate</span>
     more homogeneity for a given condition.</li>
 <li class=MsoNormal style='mso-list:l4 level1 lfo5;tab-stops:list .5in'><i
     style='mso-bidi-font-style:normal'><font size=3 face="Times New Roman"><span
     style='font-size:12.0pt;font-style:italic;mso-bidi-font-style:normal'>Standard
     Errors:</span></font></i><span style='mso-spacerun:yes'>  </span>The <span
     class=SpellE>StdDev</span> for each condition was divided by the total
     number of genes in the cluster to obtain the standard error.</li>
 <li class=MsoNormal style='mso-list:l4 level1 lfo5;tab-stops:list .5in'><i
     style='mso-bidi-font-style:normal'><font size=3 face="Times New Roman"><span
     style='font-size:12.0pt;font-style:italic;mso-bidi-font-style:normal'>Count
     of positive transcripts:</span></font></i><span style='mso-spacerun:yes'> 
     </span>A count of how many transcripts were up-regulated for each
     condition (expression &gt; 0).</li>
 <li class=MsoNormal style='mso-list:l4 level1 lfo5;tab-stops:list .5in'><i
     style='mso-bidi-font-style:normal'><font size=3 face="Times New Roman"><span
     style='font-size:12.0pt;font-style:italic;mso-bidi-font-style:normal'>Count
     of negative transcripts:</span></font></i><span style='mso-spacerun:yes'> 
     </span>A count of how many transcripts were down-regulated for each
     condition (expression &lt; 0).</li>
 <li class=MsoNormal style='mso-list:l4 level1 lfo5;tab-stops:list .5in'><i
     style='mso-bidi-font-style:normal'><font size=3 face="Times New Roman"><span
     style='font-size:12.0pt;font-style:italic;mso-bidi-font-style:normal'>Count
     of zero transcripts:</span></font></i><span style='mso-spacerun:yes'> 
     </span>A count of how many transcripts showed no change for each condition
     (expression = 0).</li>
</ul>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><font size=5
face="Times New Roman"><span style='font-size:16.0pt;font-weight:bold;
mso-bidi-font-weight:normal'>References<span style='mso-tab-count:1'>         </span></span></font></b><a
href="#_top">top</a><b style='mso-bidi-font-weight:normal'><font size=5><span
style='font-size:16.0pt;font-weight:bold;mso-bidi-font-weight:normal'><o:p></o:p></span></font></b></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal style='margin-left:.5in;text-indent:-.5in'><!--[if supportFields]><span
style='mso-element:field-begin'></span><span
style='mso-spacerun:yes'> </span>ADDIN EN.REFLIST <span style='mso-element:
field-separator'></span><![endif]--><a name="_ENREF_1"><span style='mso-no-proof:
yes'>Smoot, M. E., K. Ono, et al. (2011). &quot;Cytoscape 2.8: new features for
data integration and network visualization.&quot; <u>Bioinformatics</u> <b
style='mso-bidi-font-weight:normal'><span style='font-weight:bold;mso-bidi-font-weight:
normal'>27</span></b>(3): 431-432.</span></a><span style='mso-no-proof:yes'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-left:.5in;text-indent:-.5in'><a name="_ENREF_2"><font
size=3 face="Times New Roman"><span style='font-size:12.0pt;mso-no-proof:yes'>Yeung,
K. Y., D. R. Haynor, et al. (2001). &quot;Validating clustering for gene
expression data.&quot; <u>Bioinformatics</u> <b style='mso-bidi-font-weight:
normal'><span style='font-weight:bold;mso-bidi-font-weight:normal'>17</span></b>(4):
309-318.</span></font></a><span style='mso-no-proof:yes'><o:p></o:p></span></p>

<p class=MsoNormal><font size=3 face="Times New Roman"><span style='font-size:
12.0pt;mso-no-proof:yes'><o:p>&nbsp;</o:p></span></font></p>

<p class=MsoNormal style='margin-left:.5in;text-indent:-.5in'><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><o:p>&nbsp;</o:p></p>

</div>

</body>

</html>
